ui/public/views/integration-docs/mobile-service-integration.html

```diff
@@ -24,6 +24,19 @@
           <button ng-switch-when="1" ng-if="serviceWritable(service)" ng-click="disableIntegration(service, key)" class="btn btn-danger" type="button">Destroy Integration</button>
           <div    ng-switch-when="2" class="spinner spinner-inline"></div>
           <span ng-if="!serviceWritable(service)">{{service.name}} is not writeable; this means that it cannot be automatically configured with an integration.</span>
+          <a href="#" data-toggle="dropdown">
```

Comment:
This is the integration description and should be visible always as shown in the wireframe pic

Reply:
hey @sedroche that's cool, I haven't seen the wireframes, can you let me know where I can take a look at those, please?